### DISCUSSION TITLE
4-3 Python Rasterio - Error: Maximum Codespace Reached <and> No Access to 16-core

### DISCUSSION THREAD

**Discussion by LenaKemp** [ua-gist604b-s24/syllabus#25] [General]
Hello,

I have two issues: 
(1) I have an error of "Maximum running codespaces reached...You'll need to stope some before you start a new codespace." I  only have five total (one from each assignment we used them for). I am also unable to stop any of them. (I do not have that drop down option to stop).
(2) I do not have any options to change my Machine type to 16-core (I only have 2 and 4-core options). 

I am stuck and unable to move forward.

<img width="932" alt="Maximum_Codespaces_Error" src="https://github.com/ua-gist604b-s24/syllabus/assets/163209046/8914d31c-a1da-4c35-a0bb-dfe014933fdb">
<img width="902" alt="No_Option_to_Stop" src="https://github.com/ua-gist604b-s24/syllabus/assets/163209046/45d18815-7e2f-4a4d-a890-f777eafadbd1">
<img width="626" alt="Missing_16-Core_Option" src="https://github.com/ua-gist604b-s24/syllabus/assets/163209046/40ff8e81-cee6-4bb7-931c-9be1c5766fd9">

**Comment by bbessick-coh** [comment]
I have the same issue as Lena with only being able to use a 2 or 4 core machine type :/

**Comment by aaryno** [comment]
Can someone confirm whether 8- and 16- are available now for the 4-3 assignment? https://classroom.github.com/a/KhowS9ih

**Comment by LenaKemp** [comment]
I have access to all cores now.
<img width="663" alt="Machine_Type_Cores" src="https://github.com/ua-gist604b-s24/syllabus/assets/163209046/20e76f21-3d75-48c6-90ee-36f95015e1c6">

**Reply by aaryno** [reply]
this is good news
